montecarlo/mcpy/plotting.py

```diff
@@ -0,0 +1,244 @@
+import os
+import numpy as np
+import matplotlib
+# matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+from matplotlib.mlab import griddata
+plt.style.use('ggplot')
+from mcpy.utils import filesafe
+import mcpy.metrics
+import itertools
+
+def plot_sweep(plot_name, sweep_keys, sweep_params, sweep_metrics, sweep_true_params, config):
+    pass
+
+def plot_metrics(plot_name, param_estimates, metric_results, true_params, config):
+    # plt.figure(figsize=(10,6))
+    for dgp_name in config['dgps'].keys(): # just one right now
+        for metric_name in config['metrics'].keys():
+            if metric_name not in config['per_plots']:
+                for method_name in config['methods'].keys():
+                    x, y = metric_results[dgp_name][method_name][metric_name]
+                    plt.plot(x, y, label=method_name)
+                plt.xlabel("X_test")
+                plt.ylabel(metric_name)
+                plt.legend()
+                plt.show()
+                # mean_rmse = np.mean(metric_results[dgp_name][method_name]['rmse'])
+                # mean_conf_length = np.mean(metric_results[dgp_name][method_name]['conf_length'])
+                # mean_coverage = np.mean(metric_results[dgp_name][method_name]['coverage'])
+                # print("means for {}: rmse: {}, conf_length: {}, coverage: {}".format(method_name, mean_rmse, mean_conf_length, mean_coverage))
```

Comment:
Delete dead code.

(Unless this is super valuable, in which case add a comment explaining why it is dead and what would need to happen for it to be enabled)